tensorboard/plugins/hparams/summary_v2.py

```diff
@@ -199,7 +204,11 @@ def _normalize_hparams(hparams):
   return result
 
 
-def _derive_session_group_name(hparams):
+def _derive_session_group_name(trial_id, hparams):
+  if trial_id is not None:
+    if not isinstance(trial_id, str):
```

Comment:
Is this the prefered way? https://six.readthedocs.io/#six.string_types

Reply:
Yes; that’s probably better. Might as well fully support Python 2.